src/main/java/de/aservo/confapi/commons/model/DirectoryCrowdBean.java

```diff
@@ -79,7 +76,7 @@
     public static class DirectoryCrowdPermissions {
 
         @XmlElement
-        private Boolean readonly;
+        private Boolean readOnly;
```

Comment:
Wenn man nur `readonly` hat ist alles klein wahrscheinlich okay, wenn es aber auch `readWrite` gäbe, wäre Camel Case schon besser